packages/server/src/fhir/operations/claimexport.ts

```diff
@@ -0,0 +1,85 @@
+import { allOk, badRequest, getReferenceString, normalizeErrorString } from '@medplum/core';
+import { FhirRequest, FhirResponse } from '@medplum/fhir-router';
+import { Binary, Claim, Media, OperationDefinition } from '@medplum/fhirtypes';
+import { getAuthenticatedContext } from '../../context';
+import { createPdf } from '../../util/pdf';
```

Comment:
Looks like this file didn't get re-added

![image](https://github.com/user-attachments/assets/4fd9241e-560b-4d93-af68-b93a2df60064)